clients/src/main/java/org/apache/kafka/common/utils/AppInfoParser.java

```diff
@@ -57,7 +56,7 @@ public static void registerAppInfo(String prefix, String id) {
             AppInfo mBean = new AppInfo();
             ManagementFactory.getPlatformMBeanServer().registerMBean(mBean, name);
         } catch (JMException e) {
-            log.warn("Error registering AppInfo mbean", e);
+            log.warn("Error registering AppInfo mbean: {}", e.getMessage());
```

Comment:
`WARN` is something to be worried about, right? If it's just informational, it should be `INFO`.